app/middlewares/validation_schema.js

```diff
@@ -0,0 +1,18 @@
+const { validationResult } = require('express-validator');
+const { badRequestError } = require('../errors');
+
+const getFormattedErrors = rawErrors =>
+  rawErrors.reduce((acc, curr) => ({ ...acc, [curr.param]: curr.msg }), {});
+
+const validationSchema = (req, res, next) => {
+  const errors = validationResult(req);
+
+  if (errors.isEmpty()) {
+    return next();
+  }
+
+  const formattedErrors = getFormattedErrors(errors.array());
+  return next(badRequestError(formattedErrors));
```

Comment:
I think `BAD_REQUEST` is too generic for this type of validations. `422` is more fitting for this and maybe use a more specific name